File: Examples/0/ProbA.java
import java.util.Scanner;

/*
 * HelloWorld.java
 *
 * Copyright (c) 2019 Lockheed Martin (LM) as an unpublished work. 
 * All Rights Reserved.
 *
 * This file does not contain any Lockheed Martin Proprietary Information.
 */

/**
 * Hello World - Problem A
 * 
 * This program can be used as a template for other CodeQuest problems.
 */
public class ProbA {

	/**
	 * Every CodeQuest solution must include a main() method; how else 
	 * will we run your code? 
	 * 
	 * If you want to test your program before sending it in for judging,
	 * remember that all input this year comes from System.in. You can use
	 * these commands from your operating system's command line, replacing
	 * "Prob00" with the name of your source code file (assuming your 
	 * source code and input file are in the same location).
	 * 
	 * javac ProbA.java
	 * java ProbA < ProbA.in.txt
	 * 
	 * If you're using an IDE like Eclipse or Netbeans, you can configure
	 * them to run your program as well.
	 * 
	 * @param args
	 */
	public static void main(String[] args) {
		// First, connect to the standard input channel
		// This "try-with-resources" block will automatically close
		// the Scanner when we're done using it (even if your program
		// throws an Exception).
		try (Scanner input = new Scanner(System.in)){
			// Next, read the number of test cases
			int testCases = Integer.parseInt(input.nextLine());
			
			// Now perform each test case. This for loop is the last part
			// common to all problems; the contents of this for loop will
			// change depending on what problem you're solving.
			for(int testcase = 0; testcase < testCases; testcase++) {
				// For Problem 0, we're just reading in text and
				// passing it directly to the standard output
				System.out.println(input.nextLine());
			}
		}
	}

}

File: Examples/0/ProbA.cs
/*
 * ProbA.cs
 *
 * Copyright (c) 2020 Lockheed Martin (LM) as an unpublished work. 
 * All Rights Reserved.
 *
 * This file does not contain any Lockheed Martin Proprietary Information.
 */
using System;

/**
 * Class CodeQuest
 * 
 * This class is used to run the HelloWorld test problem.
 * You are strongly encouraged to use this as a template
 * for use in solving other problems.
 */
class CodeQuest {
	// This is the Main method, which is the entry 
	// point for your program. Without this, your
	// program can't be run.
	static void Main(string[] args) {
		// This line reads the first line of the input and
		// converts it to an integer. All inputs start with
		// the number of test cases, so this line should be
		// at the beginning of every solution you write.
		int numTestCases = Convert.ToInt32(Console.ReadLine());
		
		// You'll want to do something for each test case.
		// This for loop controls that behavior. This line
		// is the last one that will be common to all problems.
		for(int testCase = 0; testCase < numTestCases; testCase = testCase + 1){
			// Inside the for loop, you'll want to put the handling
			// for each test case. For Hello World, we're simply
			// echoing whatever input we receive.
			string text = Console.ReadLine();
			Console.WriteLine(text);
		}
	}
}